src/frame_window_manager.cpp

```diff
@@ -90,6 +90,11 @@ auto FrameWindowManagerPolicy::place_new_window(ApplicationInfo const& app_info,
         WindowInfo window_info{};
         if (override_state(specification, window_info))
         {
+            if (!specification.output_id().is_set() && output_count > 0)
+            {
+                // Place new windows round-robin on all available outputs
+                specification.output_id() = window_count++ % output_count + 1;
```

Comment:
Readers probably both know their operator precedence well enough to avoid confusion, but it is clearer to use parentheses:

```suggestion
                specification.output_id() = (window_count++ % output_count) + 1;
```